.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,5 @@
+<!-- Love storybook? Please consider supporting our collective: 👉  https://opencollective.com/storybook/donate -->
```

Comment:
mixed feelings on this PR. at the very least i think this should go at the bottom